test/visual/message-input.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
```

Comment:
new `message-input.html` is not part of visual test suite in `test.js`, and thus screenshot verification are missing.

Reply:
Done.